### PR TITLE
Update DLC-page

### PR DESCRIPTION
**Pull request by rundabollen**
Remove capitalization of first letter in months to conform to Swedish grammatical rules.
Correct typos in Zeus, Karts, Marksmen, and Apex sections.
Make Jets, Laws of War, and Tac-Ops sections be grammatically correct.

Add sections for GlobMob Cold War Germany, Contact, Art of War, and SOG Prairie Fire.

### CODE REVIEW

**Comment by rundabollen** [comment]
I realize now `DLCn` should also be changed to `DLC:n` unsure if I can adjust this after making pull request. I am not computer-man.

File: docs/arma3/README.md [hunk 1]
```diff
@@ -10,93 +10,139 @@ Till Arma 3 finns ett flertal officiella DLCs.
...
 Utökar antalet pansarfordon i spelet med tre.
-Lägger även till förbättrad simulering för pansarfordon för alla spelare.
+Lägger även till förbättrad simulering av pansarfordon för alla spelare.
 
-Denna har används i bl.a. Rolling Thunder evenemang.
+Denna har använts i bl.a. Rolling Thunder evenemang.
 
 [Steam](https://store.steampowered.com/app/798390/)
+
+
```
Comment (Dahlgren) [inline docs/arma3/README.md:104]:
```suggestion
```

File: docs/arma3/README.md [hunk 2]
```diff
@@ -10,93 +10,139 @@ Till Arma 3 finns ett flertal officiella DLCs.
...
+[Steam](https://store.steampowered.com/app/1021790/)
+
+### Art of War
+
+Släpptes 23 februari 2021
+
+Innehåller förutom ett konstgalleri, med Armarelaterad konst, ett klädesplagg och väskor.
+
+
```
Comment (Dahlgren) [inline docs/arma3/README.md:133]:
```suggestion
```

File: docs/arma3/README.md [hunk 3]
```diff
@@ -10,93 +10,138 @@ Till Arma 3 finns ett flertal officiella DLCs.
...
+Detta DLC kommer att bli gratis vid "slutet av 2021".
+
+[Steam](https://store.steampowered.com/app/1325500)
+
+### S.O.G. Prairie Fire
+
+Släpptes 6 maj 2021
+
+Lägger till en en singleplayerkampanj och flera Coopuppdrag som utspelar sig under Vietnamkriget.
```
Comment (Dahlgren) [inline docs/arma3/README.md:143]:
```suggestion
Lägger till en enspelarkampanj och flera samspelsuppdrag som utspelar sig under Vietnamkriget.
```

**Pull request merged**